drafts/samesite-cookies.adoc

```diff
@@ -1,134 +1,157 @@
-additional_authors:
-- name: Paul Nicolucci
-  github: https://github.com/pnicolucci
-  image: https://avatars1.githubusercontent.com/u/29307632
+seo-title: Setting the SameSite attribute on cookies with Open Liberty
+seo-description: You can now specify the `SameSite` cookie attribute for your application-defined cookies, as well as for the session and security cookies, in your server configuration and minimize application-level changes. The `SameSite` cookie attribute is used by web browsers to determine if a particular cookie should be sent with a request.
+blog_description: You can now specify the `SameSite` cookie attribute for your application-defined cookies, as well as for the session and security cookies, in your server configuration and minimize application-level changes. The `SameSite` cookie attribute is used by web browsers to determine if a particular cookie should be sent with a request.
 ---
-= Setting the SameSite Attribute on Cookies
-Paul Nicolucci https://twitter.com/PaulNicolucci
+= Setting the SameSite attribute on cookies with Open Liberty
+Paul Nicolucci <https://github.com/pnicolucci>
+:imagesdir: /
+:url-prefix:
 
-The `SameSite` cookie attribute is used by web browsers to determine if a particular cookie should be sent with a request. The `SameSite` attribute has three possible values: `Lax`, `Stict`, `None`. Up until recently web browsers treated a cookie without a `SameSite` attribute as if the cookie specified a `SameSite` attribute with a value of `None`. Chrome recently changed this behavior and other web browsers will soon follow. The new web browser behavior is to treat a cookie without a `SameSite` attribute as if the cookie specified a `SameSite` attribute with a value of `Lax`. The Servlet specification does not offer any API to set the `SameSite` attribute on a https://javaee.github.io/javaee-spec/javadocs/javax/servlet/http/Cookie.html[Cookie] so there historically was only one way to set the `SameSite` attribute on cookies in Open Liberty and that was to manually write the `Set-Cookie` header and add it to the https://javaee.github.io/javaee-spec/javadocs/javax/servlet/http/HttpServletResponse.html[HttpServletResponse]. Open Liberty is now adding multiple new server configurations to allow a user to specify the `SameSite` cookie attribute for their application defined cookies as well as the session and security cookies. These new server configurations will help minimize application level changes due to the new handling of the `SameSite` attribute.
+On Open Liberty, you can now specify the `SameSite` cookie attribute for your application-defined cookies, as well as for the session and security cookies, in your server configuration and minimize application-level changes. The `SameSite` cookie attribute is used by web browsers to determine if a particular cookie should be sent with a request.
```

Comment:
Reading this again I'd like to word this a bit differently here and in the descriptions areas above. We allow the samesite attribute to be configured with wildcards and explicitly for not only application-defined,session,security cookies but for any cookie added to the response by the server as well.

Reply:
Extended the list to add ", and any cookie added to the response by the server" in that first para and in the metadata descriptions.